src/Migrate/custom/New-AzMigrateServerReplication.ps1

```diff
@@ -119,6 +127,36 @@ function New-AzMigrateServerReplication {
         # Specifies the Availability Zone to be used for VM creation.
         ${TargetAvailabilityZone},
 
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Models.Api20210210.IVMwareCbtEnableMigrationInputTargetVmtags]
+        # Specifies the tag to be used for VM creation.
+        ${VMTag},
+
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Models.Api20210210.IVMwareCbtEnableMigrationInputTargetNicTags]
+        # Specifies the tag to be used for Nic creation.
```

Comment:
NIC

Reply:
Shouldn't this be NICTags? (Plural), as we take a dictionary of tags? For all the tag inputs, same comment.

Reply:
Kept same as suggested in Resource Tagging Spec_vDraft.docx (sharepoint.com)